### PR TITLE
[OSCI] Add new OSCI author Sanjana679

### PR DESCRIPTION
**Pull request by Sanjana679**
### Description
Adds Sanjana679 as an OSCI author to the repository
 
### Issues Resolved
[List any issues this PR will resolve]

### Check List
- [ ] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

**Review by rblcoder — APPROVED** [review]
Good to hear about you, Sanjana.

File: _authors/osci_2023/Sanjana679.markdown [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+name: Sanjana Nandi 
+short_name: Sanjana
+twitter:
```
Comment (ashwin-pc) [inline _authors/osci_2023/Sanjana679.markdown:4]: Can you remove this line if you are not using it?

Comment (Sanjana679) [reply _authors/osci_2023/Sanjana679.markdown:4]: Sure! I'll do that right now

Comment (Sanjana679) [reply _authors/osci_2023/Sanjana679.markdown:4]: Oh, I got what you meant. I'll delete the twitter line.

**Review by dtaivpp — APPROVED** [review]
Looks great!

**Pull request merged**